plugins/extensions/plugins/ZeroEQPlugin.cpp

```diff
@@ -176,23 +176,15 @@ void ZeroEQPlugin::_setupHTTPServer()
     _remoteScene.registerSerializeCallback(
         std::bind( &ZeroEQPlugin::_requestScene, this ));
 
-    _httpServer->handle( _remoteTransferFunction1D );
-    _remoteTransferFunction1D.registerDeserializedCallback(
-        std::bind( &ZeroEQPlugin::_transferFunction1DUpdated, this ));
-    _remoteTransferFunction1D.registerSerializeCallback(
-        std::bind( &ZeroEQPlugin::_requestTransferFunction1D, this ));
-
     _httpServer->handle( _remoteSpikes );
     _remoteSpikes.registerDeserializedCallback(
         std::bind( &ZeroEQPlugin::_spikesUpdated, this ));
     _remoteSpikes.registerSerializeCallback(
         std::bind( &ZeroEQPlugin::_requestSpikes, this ));
 
-    _httpServer->handle( _remoteColormap );
-    _remoteColormap.registerDeserializedCallback(
-        std::bind( &ZeroEQPlugin::_colormapUpdated, this ));
-    _remoteColormap.registerSerializeCallback(
-        std::bind( &ZeroEQPlugin::_requestColormap, this ));
+    _httpServer->handlePUT( _remoteMaterialLUT );
```

Comment:
_remoteMaterialLUT is more consistent with other class member names I think. And no GET for now since the renderer is not the 'owner' of the colormap.